core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/SHACL.java

```diff
@@ -0,0 +1,822 @@
+	public static final IRI NODE_KIND_CONSTRAINT_COMPONENT;
+
+	/** sh:NodeKindConstraintComponent-nodeKind */
+	public static final IRI NODE_KIND_CONSTRAINT_COMPONENT_NODE_KIND;
+
+	/** sh:NodeShape */
+	public static final IRI NODE_SHAPE;
+
+	/** sh:NotConstraintComponent */
+	public static final IRI NOT_CONSTRAINT_COMPONENT;
+
+	/** sh:NotConstraintComponent-not */
+	public static final IRI NOT_CONSTRAINT_COMPONENT_NOT;
+
+	/** sh:OrConstraintComponent */
+	public static final IRI OR_CONSTRAINT_COMPONENT;
+
+	/** sh:OrConstraintComponent-or */
+	public static final IRI OR_CONSTRAINT_COMPONENT_OR;
+
+	/** sh:Parameter */
+	public static final IRI PARAMETER;
+
+	/** sh:Parameterizable */
+	public static final IRI PARAMETERIZABLE;
+
+	/** sh:PatternConstraintComponent */
+	public static final IRI PATTERN_CONSTRAINT_COMPONENT;
+
+	/** sh:PatternConstraintComponent-flags */
+	public static final IRI PATTERN_CONSTRAINT_COMPONENT_FLAGS;
+
+	/** sh:PatternConstraintComponent-pattern */
+	public static final IRI PATTERN_CONSTRAINT_COMPONENT_PATTERN;
+
+	/** sh:PrefixDeclaration */
+	public static final IRI PREFIX_DECLARATION;
+
+	/** sh:PropertyConstraintComponent */
+	public static final IRI PROPERTY_CONSTRAINT_COMPONENT;
+
+	/** sh:PropertyConstraintComponent-property */
+	public static final IRI PROPERTY_CONSTRAINT_COMPONENT_PROPERTY;
+
+	/** sh:PropertyGroup */
+	public static final IRI PROPERTY_GROUP;
+
+	/** sh:PropertyShape */
+	public static final IRI PROPERTY_SHAPE;
+
+	/** sh:QualifiedMaxCountConstraintComponent */
+	public static final IRI QUALIFIED_MAX_COUNT_CONSTRAINT_COMPONENT;
+
+	/** sh:QualifiedMaxCountConstraintComponent-qualifiedMaxCount */
+	public static final IRI QUALIFIED_MAX_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_MAX_COUNT;
+
+	/** sh:QualifiedMaxCountConstraintComponent-qualifiedValueShape */
+	public static final IRI QUALIFIED_MAX_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_VALUE_SHAPE;
+
+	/** sh:QualifiedMaxCountConstraintComponent-qualifiedValueShapesDisjoint */
+	public static final IRI QUALIFIED_MAX_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_VALUE_SHAPES_DISJOINT;
+
+	/** sh:QualifiedMinCountConstraintComponent */
+	public static final IRI QUALIFIED_MIN_COUNT_CONSTRAINT_COMPONENT;
+
+	/** sh:QualifiedMinCountConstraintComponent-qualifiedMinCount */
+	public static final IRI QUALIFIED_MIN_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_MIN_COUNT;
+
+	/** sh:QualifiedMinCountConstraintComponent-qualifiedValueShape */
+	public static final IRI QUALIFIED_MIN_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_VALUE_SHAPE;
+
+	/** sh:QualifiedMinCountConstraintComponent-qualifiedValueShapesDisjoint */
+	public static final IRI QUALIFIED_MIN_COUNT_CONSTRAINT_COMPONENT_QUALIFIED_VALUE_SHAPES_DISJOINT;
+
+	/** sh:ResultAnnotation */
+	public static final IRI RESULT_ANNOTATION;
+
+	/** sh:SPARQLAskExecutable */
+	public static final IRI SPARQLASK_EXECUTABLE;
+
+	/** sh:SPARQLAskValidator */
+	public static final IRI SPARQLASK_VALIDATOR;
```

Comment:
Should this be SPARQL_ASK_VALIDATOR? Same for the other ones with SPARQL in them.